pos_access_right/static/src/css/pos.css

```diff
@@ -0,0 +1,32 @@
+.pos .actionpad .button.pay.disabled-mode,
+.pos .actionpad .button.pay.disabled-mode:hover {
+    background: #c7c7c7;
+    color: #a5a1a1;
+    cursor: not-allowed;
+}
+
+.pos .actionpad .ticket-button.disabled-mode:hover {
+    background: rgb(104, 69, 95);
+    cursor: not-allowed;
+}
+
+.pos .actionpad .ticket-screen .order-row:hover {
+    background: rgb(230, 230, 230);
+    color: white;
+    cursor: not-allowed;
+}
+
+.pos .actionpad .ticket-screen .controls button.highlight {
+    background: #e2e2e2;
+    border: solid 1px #ddd;
+    color: white;
+    cursor: not-allowed;
+}
+
+.ticket-screen .pointer {
+    cursor: not-allowed;
+}
+
+.order-row .pointer .delete-button {
```

Comment:
the icon is properly displayed.
Yet when Many orders options is disabled, there is no specific icon indicating that you don' have access to this function.
Could it be possible to use the same icon as for the line :
![image](https://user-images.githubusercontent.com/1043935/119682424-2b17d300-be43-11eb-9e8d-0843f98280af.png)

Reply:
@flotho that suggestions are implemented you can re-check it.